.eslintrc.js

```diff
@@ -8,10 +8,17 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: "module"
   },
+
+  plugins: ["react"],
+
   plugins: ["prettier"],
   rules: {
     // "prettier/prettier": ["error"]
-  }
+  },
+  rules: {   
+    "react/jsx-uses-react": "error",   
+    "react/jsx-uses-vars": "error" 
+}
 };
 
 // module.exports = {
```

Comment:
remove these changes, its not part of the PR instruction

Reply:
I added the changes to resolve the “footer defined but not used” error I was having

Reply:
nice but not within the scope of the PR, remove it.

Reply:
okay